rails-time-track/app/controllers/api/projects_controller.rb

```diff
@@ -60,6 +60,23 @@ def create
       render_errors(e.message, :not_found)
     end
 
+    def update
+      if @project.update(project_params)
+        render json: @project, status: :ok
+      else
+        render json: {errors: @project.errors}
```

Comment:
What do you think about using the render_errors method which customize the render for error messages. It is in application_controller.rb

Reply:
I'll find your method useful, so I used it. Thanks.